TodoApp/Tests/TodoApp.TestContracts/Wrappers/ArgWrapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using NSubstitute;
+using TodoApp.Contracts.Models;
+using TodoApp.TestContracts.Comparers;
+
+namespace TodoApp.TestContracts.Wrappers
+{
+    public static class ArgWrapper
```

Comment:
The wrapper suffix is somewhat too generic and easily mistaken with the "real" wrappers (the ones for `DateTime` and `Guid`). I'm not saying it is necessarily a bad thing, but you might want to distinguish a bit. There are options that won't abuse the pattern-related words - such as `ArgExtented` or `ArgCustom` or `ArgToDo` (based on Your app name) or `ArgItem` (assuming the only method would be `Is` and other models would eventually get their own wrappers)…

Reply:
Same applies to the `AssertWrapper` 😉